examples/gimli/src/gimli.rs

```diff
@@ -75,7 +75,13 @@ fn squeeze_block(s: State) -> Block {
 
 fn gimli_hash_state(input: &ByteSeq, mut s: State) -> State {
     let rate = Block::length();
-    for i in 0..input.num_chunks(rate) {
+    let mut chunks = input.num_chunks(rate);
+
+    if input.len() % 16 == 0 {
+        chunks += 1;
```

Comment:
This isn't allowed in hacspec (see typechecker error `error[Hacspec]: assignment operators are not supported yet in Hacspec`).